.github/COMMIT_CONVENTION.md

```diff
@@ -0,0 +1,51 @@
+## Git Commit Message Convention
+
+> This is adapted from [Angular's commit convention](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular).
+
+#### TL;DR:
+
+Messages must be matched by the following regex:
+
+<!-- prettier-ignore -->
+```js
+/^(revert: )?(feat|fix|docs|refactor|perf|test|ci|chore|release)(\(.+\))?: .{1,50}/
+```
+
+#### Examples
+
+Appears under "Features" header:
+
+```
+feat(@material-svelte/button): add a specific new feature
+```
+
+Appears under "Bug Fixes" header:
```

Comment:
`Now you found a bug in @material-svelte/checkbox, so you fix it and add:`
(you also need to modify the code from button->checkbox)